code/datums/statistics/stat_helpers.dm

```diff
@@ -162,13 +162,13 @@
 	// 	var/datum/game_mode/mixed/mixy = ticker.mode
 	// 	for(var/datum/game_mode/GM in mixy.modes)
 	// 		mixed_gamemodes.Add(GM.name)
-
+	//
 	for(var/datum/mind/M in ticker.minds)
 		add_objectives(M)
 		if(istype(M.current, /mob/living) && !M.current.isDead())
 			add_survivor_stat(M.current)
-			if(M.special_role == "Cultist")
```

Comment:
This won't ever apply again so you can just remove the commented-out cultist check